src/ServicePulse.Host/vue/src/components/failedmessages/EditMessageHeader.vue

```diff
@@ -0,0 +1,70 @@
+<script setup>
+import { ref, watch, computed, onMounted } from "vue";
+
+const settings = defineProps({
+  header: Object,
+});
+
+let origHeaderValue = undefined;
+const header = ref(settings.header);
+
+const headerValue = computed(() => settings.header.value);
+watch(headerValue, async (newValue) => {
+  header.value.isChanged = newValue !== origHeaderValue ? true : false;
```

Comment:
```suggestion
  header.value.isChanged = newValue !== origHeaderValue;
```

Truthy will make sure this is true without the explicit true/false stuff